src/main/java/org/opensearch/ad/constant/ADCommonName.java

```diff
@@ -11,17 +11,16 @@
 
 package org.opensearch.ad.constant;
 
-import org.opensearch.ad.stats.StatNames;
+import org.opensearch.timeseries.stats.StatNames;
 
-public class CommonName {
+public class ADCommonName {
     // ======================================
     // Index name
     // ======================================
     // index name for anomaly checkpoint of each model. One model one document.
     public static final String CHECKPOINT_INDEX_NAME = ".opendistro-anomaly-checkpoints";
     // index name for anomaly detection state. Will store AD task in this index as well.
     public static final String DETECTION_STATE_INDEX = ".opendistro-anomaly-detection-state";
-    // TODO: move other index name here
```

Comment:
Any plans to change the "opendistro-anomaly" moniker? Isn't "OpenSearch" more appropriate? Can be a different PR though. OpenDistro may indicate that this is serving dated models (which is not the case AFAIK).

Reply:
To keep it backward-compatible, we will use opendistro as part of index name. For new indices (e.g., forecasting indices), we are gonna use opensearch.